software/source/server/profiles/default.py

```diff
@@ -184,4 +185,7 @@ def get_function_info(file_path):
 ALWAYS REMEMBER: You are running on a device called the O1, where the interface is entirely speech-based. Make your responses to the user VERY short. DO NOT PLAN. BE CONCISE. WRITE CODE TO RUN IT.
 Try multiple methods before saying the task is impossible. **You can do it!**
 
+Use Safari for web browsing.
+Click on the first YouTube video in the search results.
+
```

Comment:
I think both these can be taken out, Safari for being mac specific + sometimes search result might not be first on YT.